frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.scss

```
@@ -1,8 +1,11 @@
 .nb-data-consumption-card__dropdown {
   display: flex;
-  margin: 0.6em 0em;
 }
 
 .nb-data-consumption-card__dropdown-item {
   margin-right: 0.4em;
 }
+
+.nb-data-consumption-card__chart-label {
+  margin-top: 1em;
```

please use PF variable
```suggestion
  margin-top: var(--pf-global--spacer--md);
```

---

Gotcha! Thanks!